ufo/templates/layout.html

```diff
@@ -2,9 +2,7 @@
 <html lang="en">
 <head>
   <title>{% block title %}{% endblock %}</title>
-  <link rel="import" href="{{ url_for('static', filename='bower_components/polymer/polymer.html') }}" />
-  <link rel="import" href="{{ url_for('static', filename='bower_components/paper-drawer-panel/paper-drawer-panel.html') }}" />
-  <link rel="import" href="{{ url_for('static', filename='sidebar.html') }}" />
+  <link rel="import" href="{{ url_for('static', filename='vulcanized.html') }}" />
```

Comment:
Just to double-check here, vulcanized.html will contain every polymer element that we are using in the app, and even pages that don't use certain elements will load them all?  It's better than what we have before (all those separate little requests).